modules/codegen-plugin/src/smithy4s/codegen/Smithy4sCodegenPlugin.scala

```diff
@@ -174,7 +299,7 @@ object Smithy4sCodegenPlugin extends AutoPlugin {
       output = os.Path(outputPath),
       resourceOutput = os.Path(resourceOutputPath),
       skip = skipSet,
-      discoverModels = true, // we need protocol here
+      discoverModels = false, // we need protocol here
```

Comment:
```suggestion
      discoverModels = false,
```

I guess that comment can be removed now